background.js

```diff
@@ -14,8 +14,8 @@ chrome.commands.onCommand.addListener(function (command) {
         update_acc = "u/" + account_num
       }
 
-      // Google Doc 
-      const doc_regex = "https:\/\/docs.google.com\/document\/u\/[0-9].*";
+      // Google Doc, slides and sheets
+      const doc_regex = "https:\/\/docs.google.com\/document\|presentation\|spreadsheets\/u\/[0-9].*";
```

Comment:
It would probably be something like (presentation|spreadsheets)
I don't think you have used the appropriate regex
Please update and test

Reply:
Also add support for other google document types like google forms and sheets as well